### ISSUE TITLE
Removing ignored container children resets the state of the supplied 'this' object on Chrome

### ISSUE THREAD

**Issue by DaTrader** [phoenixframework/phoenix_live_view#1333]
### Environment

* Elixir version (elixir -v): 1.10.3
* Phoenix version (mix deps): 1.5.6
* Phoenix LiveView version (mix deps): 0.14.7
* NodeJS version (node -v): 12.14.7
* NPM version (npm -v): 6.13.4
* Operating system: Linux Mint
* Browsers you attempted to reproduce this bug on (the more the merrier): FF, Chrome
* Does the problem persist after removing "assets/node_modules" and trying again? Yes/no: Yes

### Actual behavior

I have this JS function that is called within the updated callback of a hook at a top-level container (let's call it element A). When certain application related conditions are met, the function removes all children in an ignored container that itself is a nested element within an element within the element A (along with many others like it). This is done in a centralized manner on purpose because of the application state that is maintained within the object provided to to the callbacks of the element A hook. 
Anyway, on FF everything works fine, but on Chrome, when the removeChild function is invoked on the ignored container element, the 'this' object gets reset to some prior state from _before_ my function was even called and that happens in the middle of its execution. In other words, the state of the 'this' object immediately after the removeChild function is invoked differs from the state immediately before the call without any of the application code altering it.
In addition to the above, if enableLatencySim is turned on, the LV "simulating xxxx ms of latency from client to server" is logged to the console in-between the messages logged by my function before and after the removeChild invocation.

I don't know if this is still the case with the more recent versions of LV, but I intend to find out as soon as I finish a feature after which the LV upgrade is scheduled.

### Expected behavior

Obviously, the state of 'this' object, or at least the state of my '_vars' object within it, should remain the same before and after invoking removeChild on an ignored container.

**Comment by josevalim** [comment]
Hi @DaTrader, please check master and if the issue persists, please provide a smaller application that reproduces the error. Thank you!

**Comment by DaTrader** [comment]
Ok. Will do.

**Comment by DaTrader** [comment]
UPDATE: Having trouble reproducing this in a separate demo app. Will provide further updates once I upgrade my project to the most recent version of LV. If the issue is still there, will take the time necessary to reproduce it in a demo app.

**Comment by DaTrader** [comment]
UPDATE: Bad news, the issue is also present in 0.15.4. Will proceed with trying to reproduce it in the demo app tomorrow.

**Comment by DaTrader** [comment]
Hi again, @josevalim 

I found the issue. TLDR: It was a flaw in my code.

Namely, Chrome fires focusout events in many cases where FF does not and as I just found out, one such case is when an input or textarea element is being removed while still focused (the element in my ignored container). It was my focusout listener code that was changing the variables in 'this' object when that happened and it had nothing to do with the workings of LV.
The reason why LV was logging the simulation messages was that the focusout code was updating the state on the server (via the pushEvent function).

Apologies for the false alarm.
